src/AdminConsole/Pages/App/Playground/Client.cshtml

```diff
@@ -27,6 +27,11 @@
             <div class="mt-8">
                 <div class="mt-6">
                     <form v-on:submit.prevent="loginSubmit" href="#" name="signin" method="post">
+                        <div class="hidden sm:block">
+                            <div class="border-b border-gray-200 -mb-px flex" aria-label="Tabs">
+                                <button v-for="tab in tabs" :key="tab.name" v-on:click.prevent="setTab(tab)" :class="[tab.current ? 'border-indigo-500 text-indigo-600' : 'border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-700', 'flex-grow border-b-2 px-1 py-4 text-center text-sm font-medium']" :aria-current="tab.current ? 'page' : undefined">{{ tab.name }}</button>
+                            </div>
+                        </div>
                         <fieldset id="registerCredentialFields" class="space-y-6">
```

Comment:
Can you add some top margin here so the text appears further away from the tab strip?